tests/e2e/util.go

```diff
@@ -1811,6 +1812,98 @@ func getWCPSessionId(hostname string, username string, password string) string {
 
 }
 
+// getk8sWindowsWorkerIPs returns the windows worker node IPs
+func getk8sWindowsWorkerIPs(ctx context.Context, client clientset.Interface, nodeName string) string {
```

Comment:
wherever you are calling this function -Make sure its expecting all worker node ip or one worker node ip

Reply:
changed the method name, as it is expecting to get windows IP for specific node which was used in pod creation